samples/unit-of-work/using-childcontainers/Version_5/Sample/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using NServiceBus;
+using Raven.Client;
+using Raven.Client.Document;
+using StructureMap;
+
+static class Program
+{
+    static void Main()
+    {
+        IDocumentStore store = new DocumentStore()
+        {
+            Url = "http://localhost:32076",
+            DefaultDatabase = "Samples.StructureMapUoW"
+        }.Initialize();
+
+     
+        BusConfiguration configuration = new BusConfiguration();
+        configuration.EndpointName("Samples.StructureMap");
+
+        #region ContainerConfiguration
+        Container container = new Container(x =>
+        {
+            x.For<IDocumentStore>().Use(store);
+            x.For<IDocumentSession>().Use(c => c.GetInstance<IDocumentStore>().OpenSession());
```

Comment:
why do you resolve IDocumentStore when u have the single instance? and why is store needed in the container? dont u only need the session?